tests/unifier_test.py

```diff
@@ -199,12 +227,193 @@ def test_insert_script_to_yml_exceptions(package_path, dir_name, file_path):
         unifier = Unifier("", None, None, None)
         unifier.package_path = package_path
         unifier.dir_name = dir_name
+        unifier.is_script_package = dir_name == 'Scripts'
         with open(file_path + ".yml", "r") as yml:
             test_yml_data = yaml.safe_load(yml)
         if dir_name == "Scripts":
             test_yml_data['script'] = 'blah'
-        elif dir_name == "Integrations":
+        else:
             test_yml_data['script']['script'] = 'blah'
 
         with pytest.raises(ValueError):
-            unifier.insert_script_to_yml(".py", "", test_yml_data)
+            unifier.insert_script_to_yml(".py", {'script': {}}, test_yml_data)
+
+
+def create_test_package(test_dir, package_name, base_yml, script_code, detailed_description='', image_file=''):
+    package_path = os.path.join(test_dir, package_name)
+
+    os.makedirs(package_path)
+    shutil.copy(base_yml, os.path.join(package_path, f'{package_name}.yml'))
+
+    with open(os.path.join(package_path, f'{package_name}.py'), 'w') as file_:
+        file_.write(script_code)
+
+    if detailed_description:
+        with open(os.path.join(package_path, f'{package_name}_description.md'), 'w') as file_:
+            file_.write(detailed_description)
+
+    if image_file:
+        shutil.copy(image_file, os.path.join(package_path, f'{package_name}_image.png'))
+
+
+class TestMergeScriptPackageToYMLIntegration:
+    def setup(self):
+        self.test_dir_path = os.path.join('tests', 'test_files', 'Unifier', 'Testing')
+        os.makedirs(self.test_dir_path)
+        self.package_name = 'SampleIntegPackage'
+        self.export_dir_path = os.path.join(self.test_dir_path, self.package_name)
+        self.expected_yml_path = os.path.join(self.test_dir_path, 'integration-SampleIntegPackage.yml')
```

Comment:
where is this file located? `integration-SampleIntegPackage.yml`

Reply:
good catch, it was missing.